tests/MongoDB.Driver.Core.TestHelpers/XunitExtensions/RequireServer.cs

```diff
@@ -162,6 +162,18 @@ public RequireServer StorageEngines(params string[] storageEngines)
             throw new SkipException($"Test skipped because storage engine is \"{actualStorageEngine}\" and not one of ({storageEnginesString}).");
         }
 
+        public RequireServer Tls(bool required = true)
+        {
+            var usingTls = CoreTestConfiguration.ConnectionString.Tls;
+            if (CoreTestConfiguration.ConnectionString.Tls == required)
+            {
+                return this;
+            }
+            throw new SkipException(
+                $"Test skipped because the connection string specifies TLS={usingTls} " +
+                $"and this test expects TLS={required}");
```

Comment:
Change `expects` to `requires`?

    $"and this test requires TLS={required}");

Reply:
Done